docs/plugins.md

```diff
@@ -390,3 +390,53 @@ Leaflet keeps it simple. If you can think of a feature that is not required by a
 There are no hard requirements on how to create your own plugin, but all developers are encouraged to read the recommendations in the [plugin guide](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md).
 
 Once your plugin is ready, you can submit it: just send a pull request with a new plugin file in [/docs/_plugins/](https://github.com/Leaflet/Leaflet/tree/main/docs/_plugins)to our GitHub repository.
+
+<script>
+function loadRepoData() {
+	const tables = document.querySelectorAll('.plugins');
+	tables.forEach((table) => {
+		const rows = table.querySelectorAll('tr');
```

Comment:
`const repos = document.querySelectorAll('.plugins table tr .repo-data')`

No need for extra checks - if stuff doesn't exist, then it'll return a zero-sized set.

Reply:
👍